File: src/test/java/com/tom/PersonTest.java
package com.tom;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class PersonTest {
    @Test
    public void bmiTest(){
        Person person = new Person(55.2f,1.65f);
        Assertions.assertEquals(55.2f/(1.65f*1.65f),person.bmi());
    }
}

File: src/test/kotlin/com/tom/PersonTest.kt
package com.tom.kotlin

import com.tom.Human
import com.tom.Person
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class PersonTest {
    @Test
    fun humonBmiTest(){
        val humon = Human("Leo",95.4f,1.86f)
        Assertions.assertEquals(95.4f/(1.86f*1.86f),humon.bmi())
    }

    @Test
    fun personBmiTest(){
        val person = Person(45.7f,1.554f)
        Assertions.assertEquals(45.7f/(1.554f*1.554f),person.bmi())
    }
}